Automattic/wp-calypso
# AI Logo generator: Add feedback button on footer

Closes #85778

## Proposed Changes

* Adds a feedback button with same link as the assistant block sidebar
* Adds the AI icon
* Fixes the logo presenter style for mobile

## Testing Instructions



* Go to the generator modal
* Find and click on the link
* Check that you're redirected to the feedback form

## Pre-merge Checklist
